test_haystack/whoosh_tests/test_whoosh_backend.py

```diff
@@ -752,6 +757,11 @@ def test_scoring(self):
             ["0.40", "0.40", "0.40"],
         )
 
+    def test_analyzed_fields(self):  # TODO: rename to test_analyzed_fields
+        self.sb.update(self.wmmi, self.sample_objs)
+        results = self.whoosh_search("name_analyzed:daniel")
```

Comment:
This test doesn't appear to exercise the analyzer – at first glance it looks like we should be able to test it by adding some digits which should be stripped by the substitution filter:

```suggestion
        results = self.whoosh_search("name_analyzed:12345daniel67890")
```